war/src/main/less/base/typography.less

```diff
@@ -9,6 +9,11 @@ p {
   color: @text-color;
 }
 
+button {
+  font-family: inherit;
+  font-size: inherit;
```

Comment:
Did you verify that this `font-size` has lower preference than the one specified on `buttons.less`. If the  `inherit` takes preference, buttons will have whichever font-size their parent has.

Reply:
buttons.less has higher specificity by using:
`.yui-button button,` so should be fine, can probably remove it from here since it's set there as well?

Reply:
Nah, we can leave it in case someone tries to use unstyled buttons for some reason.